atom-jobs/build-common.groovy

```diff
@@ -795,8 +795,15 @@ def packageBinary() {
         sh """
         cd ${TARGET}
         tar --exclude=${TARGET}.tar.gz -czvf ${TARGET}.tar.gz *
-        curl -F ${OUTPUT_BINARY}=@${TARGET}.tar.gz ${FILE_SERVER_URL}/upload
+        curl -F ${OUTPUT_BINARY}=@${TARGET}.tar.gz ${FILE_SFERVER_URL}/upload
         """
+        if (PRODUCT == "enterprise-plugin") {
+            sh """
+                upload.py ${TARGET}.tar.gz ${TARGET}.tar.gz
+                aws s3 cp ${TARGET}.tar.gz s3://download.pingcap.org/${TARGET}.tar.gz --acl public-read
```

Comment:
upload plugin to s3 in build-common is not correct. maybe we should do this on ga release pipeline.